webapp/packages/core-utils/src/getMIME.ts

```diff
@@ -6,9 +6,9 @@
  * you may not use this file except in compliance with the License.
  */
 
-export function getMIME(binary: string): string | null {
+export function getMIME(binary: string): string {
   if (binary.length === 0) {
-    return null;
+    return 'application/octet-stream';
```

Comment:
check the conversation tab. this is the reason why image presentation now broken for blobs which are not images

Reply:
fixed